sql/flow_test.go

```diff
@@ -54,9 +57,44 @@ func TestCommonDockerUtilSuccess(t *testing.T) {
 		mockDockerBinder.On("ContainerLogs", mock.Anything, mock.Anything, mock.Anything).Return(sampleLog, nil)
 		return mockDockerBinder, nil
 	}
+	PrintBuildingSteps = func(r io.Reader) error {
+		return nil
+	}
```

Comment:
Can we define this nil return function as some mockedPrintBuildingSteps function and use that here? Because the same 3 lines are repeated in most test cases.